srml/balances/src/lib.rs

```diff
@@ -464,6 +464,23 @@ decl_module! {
 			let dest = T::Lookup::lookup(dest)?;
 			<Self as Currency<_>>::transfer(&source, &dest, value)?;
 		}
+
+		/// Transfer some liquid free balance to another account, while checking that the transfer
+		/// will not kill the account.
+		///
+		/// 99% of the time you want `transfer` instead.
```

Comment:
I feel this should more directly point to transfer function or repeat the weight paragraph.
```
		/// Same as `transfer` call, transfer some liquid free balance to another account, but only if from account is kept alive.
        ///
		/// 99% of the time you want `transfer` instead.